packages/data/addon/serializers/dimensions/elide.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import NaviDimensionSerializer from './interface';
+import NaviDimensionModel from '../../models/navi-dimension';
+import { DimensionColumn } from '../../adapters/dimensions/interface';
+import { AsyncQueryResponse } from 'navi-data/adapters/facts/interface';
+import EmberObject from '@ember/object';
+
+type ResponseEdge = {
+  node: Dict<string>;
+};
+
+export default class ElideDimensionSerializer extends EmberObject implements NaviDimensionSerializer {
+  normalize(dimension: DimensionColumn, rawPayload: AsyncQueryResponse): NaviDimensionModel[] {
+    const responseStr = rawPayload?.asyncQuery.edges[0].node.result?.responseBody;
+    const { id, tableId } = dimension.columnMetadata;
+    const match = new RegExp(`${tableId}\\.(.*)`).exec(id); // Remove table id from start of dimension e.g. tableA.dim1 => dim1
+    const dimensionName = match ? match[1] : id;
+
+    if (responseStr) {
+      const response = JSON.parse(responseStr);
+      return response.data[tableId as string].edges.map((edge: ResponseEdge) =>
+        NaviDimensionModel.create({
+          value: edge.node[dimensionName], //TODO: Use canonicalized dimension name when Elide supports it
```

Comment:
Can we use this

https://github.com/yahoo/navi/blob/72021ee34de8b8848f228c03d9351b585191c5e4/packages/data/addon/adapters/facts/elide.ts#L34

Reply:
That function is specifically for formatting for the query string. I think we should use the `canonicalizeMetric` function once it is generic

Reply:
`canonicalizeMetric` is the Navi normalized form. We need to take a NaviColumn and get an Elide field (which what `getElideField` does) out of the response.

Reply:
Right but we don't know that the format for querying a parameterized field is the same as what will be returned in the response since parameterized fields aren't really supported in Elide yet. I suggested before that we meet with the WS team to make sure the format is understood by both sides.

Reply:
This isn't currently on Elide's roadmap, so I don't think we can rely on a getting a spec.  This is how we did the fact serializer:

https://github.com/yahoo/navi/blob/a3df6a181a7c3dd9527487e8c310f1fadf70d6dd/packages/data/addon/serializers/facts/elide.ts#L24

Ultimately, this is about keeping the code DRY.

Reply:
Okay that's fine for now. I guess we'll have to revisit this either way